contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from future import standard_library
+standard_library.install_aliases()
+import json, urllib.request, urllib.parse, urllib.error
+class SpotifyHandler(object):
+    '''
+    This plugin will allow users to directly search
+    for song, artists, and albums using the Spotify API.
+    It can be called using `@spotify song/artist/album`
```

Comment:
@trueskawka 
1. We could, but I haven't put much thought into that yet. I still need to figure out how to implement that.
2. Nope, even Playlist is available as a end point, I just realised that after reading the official docs.
I'll get working on both ASAP, could I get a day extension to my task? I think this will take me some time.